src/Service.Tests/Authorization/AuthorizationHelpers.cs

```diff
@@ -69,7 +69,8 @@ public static AuthorizationResolver InitAuthorizationResolver(RuntimeConfig runt
             HashSet<string>? excludedCols = null,
             string? databasePolicy = null,
             string? requestPolicy = null,
-            string authProvider = "AppService"
+            string authProvider = "AppService",
```

Comment:
remove the "/// <returns></returns>" from the param comment

Reply:
Adding the missing parameters summary.